app/components/cases/cases-filter/component.js

```diff
@@ -33,13 +36,19 @@ export default Component.extend({
 	}).restartable(),
 
 	actions: {
-		selectDateFrom(date) {
-			this.set('dateFrom', date);
-			this.searchTask.perform();
-		},
-		selectDateTo(date) {
-			this.set('dateTo', date);
-			this.searchTask.perform();
-		}
+    selectDateFrom(date) {
+      this.set('dateFrom', date);
+      this.searchTask.perform();
+    },
+    selectDateTo(date) {
+      this.set('dateTo', date);
+      this.searchTask.perform();
+    },
+    initTrigger() {
+      this.set('popoverShown', true);
+    },
+    closeModal() {
```

Comment:
It's not a modal, I would suggest against naming it that way. Use something like `closePopover` or `closeLegend` instead.

Reply:
Updated